.github/workflows/CI.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           - '3.10'
           - '3.11'
           - '3.12'
+          - '3.13-dev'
```

Comment:
You should use '3.13' and set allow-prereleases: true instead. That won't need to be updated in the future.